Core/Data.EF/ApplicationDbContext.cs

```diff
@@ -39,6 +39,19 @@ public class ApplicationDbContext : DbContext
             _modelBuilders = modelBuilders;
         }
 
+        public ApplicationDbContext(DbContextOptions options,
```

Comment:
Avoid duplicating code. You can derive the construct from the existing one, like this

public ApplicationDbContext(DbContextOptions options,
            IUserContext userContext,
            IEnumerable<IModelBuilderService> modelBuilders) : this(options, userContext, null, modelBuilders)
        {
        }

Reply:
Fixed